pilot/pkg/config/kube/gateway/testdata/eastwest-namedport.yaml

```diff
@@ -0,0 +1,56 @@
+apiVersion: gateway.networking.k8s.io/v1alpha2
+kind: Gateway
+metadata:
+  name: eastwestgateway
+  namespace: istio-system
+  labels:
+    topology.istio.io/network: "network-1"
+spec:
+  gatewayClassName: istio
+  listeners:
+  - name: istiod-grpc
+    port: 15012
+    protocol: TLS
+    tls:
+      mode: Passthrough
+  - name: istiod-webhook
+    port: 15017
+    protocol: TLS
+    tls:
+      mode: Passthrough
+  - name: tls
```

Comment:
Not sure I see the difference with the previous test ( labelport ). 

Is this intended as a negative test ? I think the name of the 15443 port should be very explicit and used in the check, not 'tls'

Reply:
Tests we can infer from name `tls-passthrough`